bridges/otelzap/README.md

```diff
@@ -0,0 +1,5 @@
+# OpenTelemetry Zap Log Bridge
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/contrib/bridges/otelzap.svg)](https://pkg.go.dev/go.opentelemetry.io/contrib/bridges/otelzap)
+
+This package implements a [Log Bridge](https://opentelemetry.io/docs/specs/otel/logs/bridge-api/) for [zap](https://github.com/uber-go/zap) logger. This bridge can be used to send logs emitted by application using zap to an OpenTelemetry compatible endpoint through the [OpenTelemetry Logs Bridge API](https://pkg.go.dev/go.opentelemetry.io/otel/log).
```

Comment:
This overlaps with the [package documentation](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/579e7ea321eca94587e2d7a45b6e817b8d110f0c/bridges/otelzap/core.go#L4-L5).

```suggestion
```

Reply:
fixed